docs/connectivity/Agent.mdx

```diff
@@ -0,0 +1,275 @@
+---
+id: Agent
+title: Instrument Agents
+displayed_sidebar: webUiSidebar
+---
+
+import { PlusCircleOutlined, 
+         SelectOutlined,
+         DiffOutlined
+       } from '@ant-design/icons';
+
+The Ganymede Agent is an executable that can be installed on lab instruments running Windows or Linux to capture instrument data systematically.  The agent can
+
+- filter and designate how to process data
+- interact with connected instruments
+- upload files into Ganymede Cloud
+- initiate data processing pipelines
+
+By installing Agents, your lab machines become an integrated part of the Ganymede system, enhancing the data management and processing capabilities in your lab.
+
+## What does an Agent consist of?
+
+Each Agent has 3 questions to answer:
+
+- What files are relevant for capture?
+- How should files (from potentially different instruments) be standardized for consumption (by a common flow)?
+- Where should the standardized file be directed?
+
+These questions are answered by the 3 components of an Agent, which function in sequence:
+
+- A **Listener**, which looks for relevant, new files appearing on an instrument computer
+- A **Normalizer**, which is a user-defined Python snippet that normalizes captured files into a desired, standardized format
+- An **Action**, which is the action to be performed with the normalized data.  The Agent can be configured to upload a data file, start a flow, or complete execution without communicating with Ganymede Cloud.
```

Comment:
Should rephrase the last part of this, to separate out when Agent takes a Ganymede action vs when it stops without action.